CHANGELOG.md

```diff
@@ -8,6 +8,12 @@ here: [Community Features](https://github.com/SynthstromAudible/DelugeFirmware/b
 
 ### User Interface
 
+-  Added a new default setting that controls which playback mode new slices of a kit will get. 
+    - To change the setting, press `SHIFT` + `SELECT` : `MENU > DEFAULTS > SAMPLE SLICE MODE`.
+    - every new slice in a kit using the slicer will now get one of the modes by default
+      -  `CUT`, `ONCE`, `LOOP`, `STRETCH`
+    - the setting is persistent after reboot
+    - if a kit slice is shorter then 2s, the slicer will automatically switch to `ONCE` (default behaviour)
```

Comment:
```suggestion
```

Detailed description of the changes should live in `community_features`.

Reply:
alright!